README.rst

```diff
@@ -69,23 +69,112 @@ Windows
+see https://cloud.google.com/docs/authentication/production. Here is an example of a client
+using ADC to authenticate:
+
+.. code-block:: python
+
+    from google.cloud import compute_v1
+
+    networks_client = compute_v1.NetworksClient()
+    for network in networks_client.list(project='YOUR_PROJECT'):
+        print(network)
+
+
+You can use a file with credentials to authenticate and authorize, such as a JSON key
+file associated with a Google service account. Service Account keys can be created and
+downloaded from https://console.cloud.google.com/iam-admin/serviceaccounts. The library used
+to create the credentials objects is ``google-auth``. This example uses the Networks Client, but
+the same steps apply to the other clients in this package.
+Example:
+
+.. code-block:: python
+
+    from google.oauth2 import service_account
+    from google.cloud import compute_v1
+
+    credentials = service_account.Credentials.from_service_account_file(
+        '/path/to/key.json')
+
+    networks_client = compute_v1.NetworksClient(credentials=credentials)
+    for network in networks_client.list(project='YOUR_PROJECT'):
+        print(network)
+
+
+In some cases (for instance, you don't want to store secrets on disk), you can create credentials
+from in-memory JSON and use the from_service_account_info method. You can also limit the use of
+your credentials only to specified scopes. Note that scopes can be found
+at https://developers.google.com/identity/protocols/oauth2/scopes. Example:
+
+.. code-block:: python
+
+    import json
+
+    from google.oauth2 import service_account
+    from google.cloud import compute_v1
+
+    json_acct_info = json.loads(function_to_get_json_creds())
+    credentials = service_account.Credentials.from_service_account_info(
+        json_acct_info)
+
+    scoped_credentials = credentials.with_scopes(
+        ['https://www.googleapis.com/auth/cloud-platform'])
+
+    networks_client = compute_v1.NetworksClient(credentials=scoped_credentials)
+    for network in networks_client.list(project='YOUR_PROJECT'):
+        print(network)
+
+Long Running Operations
+~~~~~~~~~~~~~~~~~~~~~~~
+Any LRO (Long Running Operation), like the many ``insert()`` operations, can be handled using
+an ``ExtendedOperation`` object that is returned when a long running operation is started.
+
+You can wait for the completion of an operation using ``result()`` method. This method accepts
+a ``timeout`` argument, specifying how long you want your process to wait for completion of the
+operation (in seconds). When the call to ``result()`` times out, the operation is not automatically
+cancelled. To cancel an operation you can use the ``cancel()`` method. You can query the operation
+at any time to check if it's complete with ``done()``.
+
+A sample method to handle LROs featuring error and warning reporting can be found in the Python
+code samples repository: [add link here]
```

Comment:
Link missing

Reply:
Waiting for https://github.com/GoogleCloudPlatform/python-docs-samples/pull/9104

Reply:
Link added.